app/helpers/rate_limit_checker_helper.rb

```diff
@@ -2,9 +2,14 @@ module RateLimitCheckerHelper
   CONFIGURABLE_RATES = {
     rate_limit_article_update: {
       min: 1,
-      placeholder: 150,
+      placeholder: 30,
```

Comment:
I recently lowered this in our system to 30 which I think is much more practical so I changed the placeholder and default.